### PR TITLE
Feature/rc https

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@stulees @tejohnso please review. cheers

### CODE REVIEW

File: src/js/rise-cache.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ RiseVision.Common = RiseVision.Common || {};
 RiseVision.Common.RiseCache = (function () {
   "use strict";
 
-  var BASE_CACHE_URL = "http://localhost:9494/";
+  var BASE_CACHE_URL = "https://localhost:9494/";
```
Comment (stulees) [inline src/js/rise-cache.js]: @rodrigopavezi Did the RC v1 allow for https? If not, won't this always fail for Widgets running on v1 or v2 players?

Comment (rodrigopavezi) [inline src/js/rise-cache.js]: @stulees good catch. I will make the change. cheers

**Comment by rodrigopavezi** [comment]
@stulees I have made the change. please review. cheers

File: src/js/rise-cache.js [hunk 2]
```diff
@@ -44,10 +45,21 @@ RiseVision.Common.RiseCache = (function () {
             // call ping again so correct ping URL is used for Rise Cache V2
             return self.ping(callback);
           } else {
-            console.debug("Rise Cache is not running");
-            _isCacheRunning = false;
 
-            callback(false, null);
+            if ( _isHttp ) {
```
Comment (stulees) [inline src/js/rise-cache.js]: @rodrigopavezi What is the status of a failed `http://localhost:9494` ping? If it's 404, this code will not execute and instead will be in an infinite loop of executing the block above.

Comment (rodrigopavezi) [inline src/js/rise-cache.js]: @stulees good question, it will be something else but 404. It returns ERR_EMPTY_RESPONSE. I tested myself. cheers

Comment (stulees) [inline src/js/rise-cache.js]: @rodrigopavezi Ok cool, thanks

**Pull request merged**